src/main/resources/l10n/JabRef_fa.properties

```diff
@@ -1149,7 +1149,7 @@ Settings=
 
 Shortcut=
 
-Show/edit_BibTeX_source=
+Show/edit_%0_source=Montrer/�diter_le_source_BibTeX
```

Comment:
Indeed, I copied this into the wrong language file... Thanks a lot for spotting!

I will fix this directly on master, but I am actually not sure what to do with the UTF-8 problem. I copied the string from the french localization file. Do encodings differ between the localization files?